packages/backend/migrations/20241205153035_seed_operator_tenant.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.raw(`
+        INSERT INTO "tenants" ("id", "apiSecret") 
+        VALUES ('${process.env['OPERATOR_TENANT_ID']}', '${process.env['OPERATOR_TENANT_SECRET']}')
```

Comment:
technically, we don't need to store the `OPERATOR_TENANT_SECRET` in the DB if we have it in the process, thoughts?

Reply:
How does this behave when running manually? I guess it errors unless you've set them in the process. Not sure there is much to do about this. Maybe throw an error to make it more obvious?

Reply:
Operator email must be inserted as well since it is required.

Reply:
@sanducb Nathan [updated](https://github.com/interledger/rafiki/pull/3140/commits/3625938cdb5fa3ce8d8ab2b06da0e68fd30357b4) his PR to make some fields including email optional